### ISSUE TITLE
Support Data Collector: warns about utf8mb4

### ISSUE THREAD

**Issue by bschmalhofer** [RotherOSS/otobo#200]
These warnings are not warranted:

```
Zeichensatz des Datenbank-Servers (utf8mb4)

This character set is not yet supported, please see https://bugs.otrs.org/show_bug.cgi?id=12361. Please convert your database to the character set 'utf8'.
Problem Zeichensatz der Tabellen (acl, acl_sync, article, article_data_mime, article_data_mime_attachment, article_data_mime_plain, article_data_mime_send_error, article_data_otobo_chat, article_flag, article_search_index, article_sender_type, auto_response, auto_response_type, calendar, calendar_appointment, calendar_appointment_ticket, cloud_service_config, communication_channel, communication_log, communication_log_obj_lookup, communication_log_object, communication_log_object_entry, customer_company, customer_preferences, customer_user, customer_user_customer, dynamic_field, dynamic_field_obj_id_name, dynamic_field_value, follow_up_possible, form_draft, generic_agent_jobs, gi_debugger_entry, gi_debugger_entry_content, gi_webservice_config, gi_webservice_config_history, group_customer, group_customer_user, group_role, group_user, groups_table, link_object, link_relation, link_state, link_type, mail_account, mail_queue, notification_event, notification_event_item, notification_event_message, package_repository, personal_queues, personal_services, pm_activity, pm_activity_dialog, pm_entity_sync, pm_process, pm_transition, pm_transition_action, postmaster_filter, process_id, queue, queue_auto_response, queue_preferences, queue_standard_template, role_user, roles, salutation, scheduler_future_task, scheduler_recurrent_task, scheduler_task, search_profile, service, service_customer_user, service_preferences, service_sla, sessions, signature, sla, sla_preferences, smime_signer_cert_relations, standard_attachment, standard_template, standard_template_attachment, sysconfig_default, sysconfig_default_version, sysconfig_deployment, sysconfig_deployment_lock, sysconfig_modified, sysconfig_modified_version, system_address, system_data, system_maintenance, ticket, ticket_flag, ticket_history, ticket_history_type, ticket_index, ticket_lock_index, ticket_lock_type, ticket_loop_protection, ticket_number_counter, ticket_priority, ticket_state, ticket_state_type, ticket_type, ticket_watcher, time_accounting, user_preferences, users, valid, virtual_fs, virtual_fs_db, virtual_fs_preferences, web_upload_cache, xml_storage)

Es wurden Tabellen gefunden, die nicht 'utf8' als Zeichensatz haben. 
```

**Comment by svenoe** [comment]
Fixed with c23c33e2c50132e6d3c26d0276dc7d01dbb038cb ?

**Comment by bschmalhofer** [comment]
Is already fixed in the newest version.
